app/src/main/java/org/wikipedia/views/ImagePreviewDialog.kt

```diff
@@ -0,0 +1,143 @@
+package org.wikipedia.views
+
+import android.content.DialogInterface
+import android.content.Intent
+import android.graphics.drawable.Animatable
+import android.net.Uri
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.annotation.NonNull
+import androidx.annotation.Nullable
+import androidx.appcompat.widget.PopupMenu
+import com.facebook.drawee.backends.pipeline.Fresco
+import com.facebook.drawee.controller.BaseControllerListener
+import kotlinx.android.synthetic.main.dialog_image_preview.*
+import org.wikipedia.Constants
+import org.wikipedia.R
+import org.wikipedia.WikipediaApp
+import org.wikipedia.gallery.GalleryActivity
+import org.wikipedia.gallery.ImageInfo
+import org.wikipedia.page.ExtendedBottomSheetDialogFragment
+import org.wikipedia.util.L10nUtil.setConditionalLayoutDirection
+import org.wikipedia.util.StringUtil
+import org.wikipedia.util.UriUtil
+import org.wikipedia.util.log.L
+
+
+class ImagePreviewDialog : ExtendedBottomSheetDialogFragment(), DialogInterface.OnDismissListener {
+
+
+    private var fileName: String? = null
+    private lateinit var imageInfo: ImageInfo
+
+    private val menuListener = PopupMenu.OnMenuItemClickListener { item ->
+        false
```

Comment:
Empty menu actions?

Reply:
oh I have a question for Robin on the ticket... I couldn't see what options need to be there..he has answered but I still have questions as that option is already on the screen... so will be done in a separate pr.

Reply:
Robin confirmed that there shouldn't be any menu option. So removed.